cmd/buildah/list.go

```diff
@@ -38,19 +50,76 @@ func listCmd(c *cli.Context) error {
 		quiet = c.Bool("quiet")
 	}
 
-	builders, err := openBuilders(store)
-	if err != nil {
-		return fmt.Errorf("error reading build containers: %v", err)
+	truncate := true
+	if c.IsSet("no-trunc") {
+		truncate = !c.Bool("no-trunc")
 	}
-	if len(builders) > 0 && !quiet {
-		fmt.Printf("%-64s %-64s %-10s %s\n", "CONTAINER ID", "IMAGE ID", "IMAGE NAME", "CONTAINER NAME")
+
+	containers := true
+	if c.IsSet("images") {
+		containers = !c.Bool("images")
 	}
-	for _, builder := range builders {
-		if builder.FromImage == "" {
-			builder.FromImage = buildah.BaseImageFakeName
-		}
-		fmt.Printf("%-64s %-64s %-10s %s\n", builder.ContainerID, builder.FromImageID, builder.FromImage, builder.Container)
+
+	images := true
+	if c.IsSet("containers") {
+		images = !c.Bool("containers")
 	}
 
+	if images {
+		images, err := store.Images()
+		if err != nil {
+			return err
+		}
+
+		if len(images) > 0 && !quiet {
+			fmt.Printf("\nIMAGES\n\n")
+			if truncate {
+				fmt.Printf("%-12s %-64s\n", "IMAGE ID", "IMAGE NAME")
```

Comment:
Since we are trying to stay within 80 chars, I thought we would just do 64, also the Name is somewhat important while the ID is pretty much only used by the computer.
We have a bug in this patch in that you can not use the short name for delete operations.